### ISSUE TITLE
[Bug] 

### ISSUE THREAD

**Issue by BylethC** [lobehub/sd-webui-lobe-theme#412]
### 💻 系统环境 | Operating System

Windows

### 🌐 浏览器 | Browser

Edge

### 📦 SD Wbui 版本或提交 | Version or Commit

5ef669d

### 📦 Lobe Theme 版本或提交 | Version or Commit

ea600a4

### 🐛 问题描述 | Bug Description

开启lobe主题插件后，几乎导致webui的模型预览图功能完全失效，永远加载不出来

只要关闭lobe插件，恢复到webui默认的gradio样式后，就会一切正常！

![image](https://github.com/lobehub/sd-webui-lobe-theme/assets/46842736/0835196c-0e10-4c3f-b2a6-54978936e239)

### 📷 复现步骤 | Recurrence Steps

开启lobe主题插件，尝试使用webui的模型预览图功能即可复现问题

**Comment by canisminor1990** [comment]
排查方法，将 webui 和 lobe theme 更新到最新，如果依然有问题，尝试排查↓是否和其他插件存在冲突

**Comment by canisminor1990** [comment]
https://github.com/lobehub/sd-webui-lobe-theme/issues/410

**Comment by antorio** [comment]
webui already latest, lobe theme already latest, no conflict log on console whatsoever. This infinite loading problem is still going on. And why the image is not just cache from webui? seems to new loading everytime it opens. 

How to solve this? the source code already minified

Or please just take off the loading layer altogether if u will

**Comment by kukudoe** [comment]
是不是和lora-prompt-tool冲突了

**Comment by jinjin327** [comment]
是的，最新3.08版本无限加载问题依然存在

**Comment by Kryptonic83** [comment]
I'm having same problem in Edge browser as well, strange.

**Comment by lllllllllllllllllllllllll** [comment]
I'm also having a strange version of this problem. In chrome browsers it doesn't load at all like the others said, but in Firefox it shows the model cards with the loading overlay that never goes away. I can select and use them just fine, the only issue is it's darkened and loading number keeps increasing and never goes away. 

Is there a way to just disable the loading overlay since that's the obvious bugged part?

![Stable_Diffusion_—_Mozilla_Firefox_NLy6XYPN8U](https://github.com/lobehub/sd-webui-lobe-theme/assets/11816587/65a4f4bb-4232-4f70-9541-c61dcad66199)

**Comment by antorio** [comment]
is there anybody know how to solve this? or have any suspect to where the problem lies? the last update seems do nothing to this problem

**Comment by snowyu** [comment]
@canisminor1990  同上, 所有的预览全部卡住扩展页面也卡住, 今天刚clone的最新的版本,  这也导致:

1. generate失效,无法创建任何图.
2. ctrl+c停止sd, 然后从sd的extensions目录中移除sd-webui-lobe-theme,重启sd, 依旧卡死, generate依然失效,无法创建任何图.
    * 必须在扩展选项中点击"Apply and restart UI" 才能恢复正常.

**Comment by Pitpe12** [comment]
Try removing all Extra networks (Lora, Lycoris) from the folders and only keeping a few you want to use right now, it should remove the infinite loading. It did in my machine, at least. I think the Webui cannot work with more than 100-200 extra networks in the folders, I think it depends on each machine (or maybe it's something completely unrelated to the machine's performance I cannot say), but that's where mine started to cause infinite loading.

**Comment by antorio** [comment]
> Try removing all Extra networks (Lora, Lycoris) from the folders and only keeping a few you want to use right now, it should remove the infinite loading. It did in my machine, at least. I think the Webui cannot work with more than 100-200 extra networks in the folders, I think it depends on each machine (or maybe it's something completely unrelated to the machine's performance I cannot say), but that's where mine started to cause infinite loading.

@Pitpe12 I don't think we can blame this to A1111 webui, because **without Lobe Theme** they all _smoothly no error nor the infinite loading_. Besides, I don't use extra network that many, only have about 20-30 of them. And I have tried on three different machines.

**Comment by Pitpe12** [comment]
> > Try removing all Extra networks (Lora, Lycoris) from the folders and only keeping a few you want to use right now, it should remove the infinite loading. It did in my machine, at least. I think the Webui cannot work with more than 100-200 extra networks in the folders, I think it depends on each machine (or maybe it's something completely unrelated to the machine's performance I cannot say), but that's where mine started to cause infinite loading.
> 
> @Pitpe12 I don't think we can blame this to A1111 webui, because **without Lobe Theme** they all _smoothly no error nor the infinite loading_. Besides, I don't use extra network that many, only have about 20-30 of them. And I have tried on three different machines.

Yeah, I think you are right. Went back and put well over 200 extra networks in and other than it took ~10 more seconds to fully load there was no infinite loading. But then as a test I activated the sd-webui-controlnet extension and sd-webui-openpose-editor extension (since I don't use them often I disabled them), infinite loading occurred. All I can say then it's gotta be extension conflict as mentioned above. It honestly feels like a “timeout” when it comes to loading some extensions, maybe the number of active extensions causes that? It also seems like with every extension update, lobe might not work well with different extensions each time.

**Comment by antorio** [comment]
in the meantime, I've found using **--no-gradio-queue** command parameter in webui-user.sh (or webui-user.bat in you're on windows) solved the problem. Other extension now works too. 

I think probably because lobe's preloading-code hitchhiking the websocket conn, instead of just using normal https connection, or there's no error handler when it timed-out, or no preloading time limitation, or all the above. 
When it stuck, this makes it kinda hinder other extension to works smoothly too because of the bottleneck lobe was created.
 
@Pitpe12

**Comment by Pitpe12** [comment]
@antorio Interesting fix, fortunately I have not had it happen even with various extension updates, but I'll try to use it if I ever get infinite loading again. Also, websocket being weird I guess.

**Comment by hakaserver** [comment]
The infinite loading happens to me when using chrome, but not when using firefox. Unfortunately, the civitai-shortcut extension [requires gradio queue function](https://github.com/sunnyark/civitai-shortcut/issues/38), which I use a lot.

**Comment by neokera** [comment]
I also encountered the same problem. I had to restart Webui many times before it could work normally, which greatly affected the user experience.

**Comment by chowtime3721** [comment]
I had the same problem, I noticed that this occurs on specific browser which in my case is MS Edge, however when I tried on Brave and Firefox browsers it doesn't happen similar issue as MS Edge

**Comment by chowtime3721** [comment]
I had the same problem, I noticed that this occurs on specific browser which in my case is MS Edge, I tried on Brave and Firefox browsers.. doesn't happen similar issue as MS Edge

**Comment by i-iooi-i** [comment]
遇到了相同的问题，不过这个问题更新过几版，似乎一直存在的。

![image](https://github.com/lobehub/sd-webui-lobe-theme/assets/1827701/3776abe5-deae-49d6-9cea-3b8495536b50)
![image](https://github.com/lobehub/sd-webui-lobe-theme/assets/1827701/2a953bdf-0a7a-48f2-807d-9412dc7a9eba)

重载UI好像也无法解决。

**Comment by NarakoYo** [comment]
me too

**Comment by n0kovo** [comment]
+1

**Comment by NaughtDZ** [comment]
俺也一样！

**Comment by jamsdrak** [comment]
The Issue of the infinity loading is still happening  | version: [v1.7.0] •  python: 3.10.6  •  torch: 2.0.1+cu118  •  xformers: N/A  •  gradio: 3.41.2 |

In my personalize chrome the problem keeps happening, infinite loading, in tabs Extension, Checkpoints, Lora, etc.  Especially the extension, making it annoying not being able to disable the theme. 

In incognito mode (that dosent load any chrome extension) the theme works fine. Soo Just "USE INCOGNITO MODE" or like below

PARTIAL SOLUTIONS  Add to the start bat file --no-gradio-queue  .  But apparently this disable  the civitai-shortcut extension (no sure what they meant by this)

**Comment by i-iooi-i** [comment]
似乎发现问题出在哪里了。

我使用Chrome浏览器，安装了很多Chrome扩展插件，启用 lobe theme 的时候，模型缩略图一直提示 “载入中”。
当我使用Chrome的访客模式的时候，缩略图载入正常。

应该是某些Chrome扩展插件造成的问题。
